### ISSUE TITLE
Angular: parameter for custom pagination element ignored (pagination.el)

### ISSUE THREAD

**Issue by mxlle** [nolimits4web/swiper#4110]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 6.4.6
* Platform/Target and Browser Versions: IOS, ANDROID, CHROME
* Live Link or JSFiddle/Codepen or website with isssue: [Reproduced in StackBlitz](https://stackblitz.com/edit/angular-swiper-pagination-issue?file=src%2Fapp%2Fapp.component.ts) 

### What you did
For details see stackblitz
`component.ts`:
```
swiperOptions: SwiperOptions = {
  pagination: {
    type: "progressbar",
    el: ".swiper-pagination.my-pagination-container"
  }
};
```

`component.html`:
```
<swiper [pagination]="swiperOptions.pagination">
    <ng-template swiperSlide>
      <div class="inner-slide">Hello world</div>
    </ng-template>
    <ng-template swiperSlide>
      <div class="inner-slide">How are you?</div>
    </ng-template>
    <div
      slot="container-end"
      class="swiper-pagination my-pagination-container"
    ></div>
</swiper>
```

### Expected Behavior
Pagination should be inserted in specified html node. 

### Actual Behavior
Default pagination is inserted. 

### Possibly related issue
I saw that there was a similar issue for react, which was never fully solved unfortunately: #3766

**Comment by vltansky** [comment]
fixed in `swiper@6.4.7`

**Comment by Bodjebah** [comment]
bonjour j'aimerai utiliser les swiper avec une pagination personnalise peut tu maider
